charts/reaper-operator/templates/deployment.yaml

```diff
@@ -1,18 +1,21 @@
+{{- $defaultPodSecurityCtx := "{}" }}
+{{- $defaultSecurityCtx := "readOnlyRootFilesystem: true" }}
```

Comment:
Similar to my comments for the medusa-operator chart, why redefine these here when they are already defined in values.yaml?